modules/cache.py

```diff
@@ -18,7 +18,7 @@ def __str__(self):
         return f"VideoInfo(video_id={self.video_id}, file_path={self.file_path}, size_bytes={self.size_bytes})"
 
 class Cache():
-    def __init__(self, file_path:str, max_size_bytes:int=2_000_000_000) -> None:
+    def __init__(self, file_path:str="not_available", max_size_bytes:int=2_000_000_000) -> None:
```

Comment:
do we need this extra param?

Reply:
without it, it gives me: `TypeError: Cache.__init__() missing 1 required positional argument: 'file_path'`